src/main/resources/application.properties

```diff
@@ -74,6 +74,8 @@ tse.respondent.respond.acknowledgement.rule92no.template.id=${TSE_RESPONDENT_ACK
 tse.respondent.respond.acknowledgement.rule92yes.template.id=${TSE_RESPONDENT_ACKNOWLEDGEMENT_YES_TEMPLATE_ID}
 tse.respondent.respond.notify.claimant.template.id=${TSE_RESPONDENT_RESPONSE_TEMPLATE_ID}
 tse.respondent.application.tribunal.template.id=${TSE_NEW_APPLICATION_ADMIN_TEMPLATE_ID}
+tse.respondent.reply-to-tribunal.to-tribunbal=${TSE_RESPONDENT_REPLY_TO_TRIB_TO_TRIB}
```

Comment:
I suggest adding the values currently in -dev.properties directly to here as defaults. That means those with cftlib can run it automatically and we won't need the charts.values overrides or the overrides in the preview environment. This is what we're going to transition to soon enough anyway (as it is on et-sya-api).

Reply:
updated, thanks.